modules/Sources/Features/SendFlow/SendFlow.swift

```diff
@@ -98,12 +98,16 @@ public struct SendFlow: Reducer {
         }
         public var recipient: RedactableString?
         public var memo: RedactableString?
+        public var proposal: Proposal?
+        public var feeRequired: Zatoshi {
+            proposal?.totalFeeRequired() ?? Zatoshi(0)
```

Comment:
I don't think the fee can be zero. Actually unpaid actions will be kicked out of the mempool in the next Zcashd release.

Reply:
https://zips.z.cash/zip-0317#mempool-size-limiting

Reply:
This is just to satisfy the compiler, since the proposal is modeled as optional, since it is not set until a later part of the send flow. By the time the `feeRequired` property matters, the `proposal` is non-nil